src/lib/buttons/Button.svelte

```diff
@@ -4,11 +4,12 @@
   import type { ButtonType } from '../types';
 
   const group = getContext('group');
+  let size = getContext('size');
 
   export let pill: boolean = false;
   export let outline: boolean = false;
   export let gradient: boolean = false;
-  export let size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' = group ? 'sm' : 'md';
+  // size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' ='sm';
```

Comment:
What have you done here?!!
Now you can't set the button size!
Check https://flowbite-svelte.com/components/button#Button_sizes

Reply:
oops. It's my bad.